### PR TITLE
Add rmin, fix smoothing of potentials

### PR DESCRIPTION
**Pull request by chrisjonesBSU**
This PR adds smoothing to the potentials between iterations.

### CODE REVIEW

File: msibi/utils/error_calculation.py [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 
 def calc_similarity(arr1, arr2):
+    """
+    Sum of the RDF differences, normalized by the magnitude of each RDF combined
+    Standard error?
```
Comment (jennyfothergill) [inline msibi/utils/error_calculation.py]:
```suggestion
```

File: msibi/pair.py [hunk 2]
```diff
@@ -131,6 +132,7 @@ def save_current_rdf(self, state, iteration, dr):
             The RDF bin size
         """
         rdf = self._states[state]["current_rdf"]
+        #WHAT IS THIS LINE DOING?
```
Comment (jennyfothergill) [inline msibi/pair.py]: seems like it's subtracting dr/2 from all _r_ values? I'm confused too

**Review by jennyfothergill — APPROVED** [review]
This looks really close to me! I think a little digging to answer the questions you have in the comments and then ship it! :)
I'm approving, so please merge when ready!

File: msibi/pair.py (hunk 2 above)
Comment (chrisjonesBSU) [reply msibi/pair.py]: Seems like this is performing the bin centering for the x values of the RDF, but I don't think we need to do this anymore now that we are using Freud for the RDFs. I removed it in the latest commit.

**Comment by chrisjonesBSU** [comment]
Update on this.

I think we'll need a different smoothing approach for the potentials. Using this branch, the smoothing seemed to keep shifting the curve further to the right after each iteration. Maybe we shouldn't use the savitzy_golay function for smoothing the potentials.

**Comment by chrisjonesBSU** [comment]
This is out dated now after the latest PR.

**Pull request closed without merging**